llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -1881,9 +1876,8 @@ static void insertSpills(const FrameDataInfo &FrameData, coro::Shape &Shape) {
             CurDef = LdInst->getPointerOperand();
             if (!isa<AllocaInst, LoadInst>(CurDef))
               break;
-            DIs.clear();
-            DPVs.clear();
-            findDbgDeclares(DIs, CurDef, &DPVs);
+            DIs = findDbgDeclares(Def);
+            DPVs = findDPVDeclares(Def);
```

Comment:
I think it should use `CurDef` instead of `Def`.

Reply:
Yeah, that's correct - typo slipped in here!